cssom-view-1/Overview.bs

```diff
@@ -1074,17 +1074,12 @@ result of running these steps:
     <a>caret range</a> is a collapsed
```

Comment:
Do we want to bubble the `caret range` out of the shadow roots as well? Otherwise when [getClientRect()](https://drafts.csswg.org/cssom-view/#dom-caretposition-getclientrect) is called on the `CaretPosition`, the `DOMRect`s will correspond to the position inside the shadow root but `caret node` and `caret offset` will be at the adjusted position outside the roots.

Reply:
Yes. we should bubble the `caret range` out of the shadow roots. Updated the spec in next iteration. Thank you!